draft-ietf-taps-interface.md

```diff
@@ -1987,6 +1987,22 @@ this limits the number of ConnectionReceived Events that will occur, but constra
 to the group of the Connection associated with this property. For a multi-streaming transport,
 this limits the number of allowed streams.
 
+### Private Session {#private-session}
+Name:
+: privateSession
+
+Type:
+: Boolean
+
+Default:
+: false
+
+When set to true, this property will initiate new Connections without using
+any cached information (such as session tickets or cookies) from previous
+connections that are not entangled with it. Any state generated by this
+Connection will only be shared with entangled connections. Cloned Connections
+will use saved state from within the Connection Group.
```

Comment:
For me, this sentence implies that the state generated by this new Connection will **not** be used by other Connections that are not in the same Connection Group. But maybe it's worth making this explicit.